docs/README.md

```diff
@@ -1,106 +1,109 @@
-  - [Introduction](#introduction)
-  - [Media Proxy](#media-proxy)
-    - [Key Features](#key-features)
-    - [MemIF](#memif)
-  - [MCM SDK](#mcm-sdk)
-  - [Getting Started](#getting-started)
-    - [Prerequisites](#prerequisites)
-    - [Basic Installation](#basic-installation)
-  - [Usage](#usage)
-  - [Known Issues](#known-issues)
-  - [Support](#support)
-  - [Note](#note)
+![Media-Communication-Mesh](./img/media-communications-mesh-proto-1.webp)
 
 ## Introduction
 
 The Media Communications Mesh (MCM) enables efficient, low-latency media transport for media microservices for Edge, Edge-to-Cloud, and both private and public Cloud environments. The framework creates a secure, standards-based media data plane for inter-microservices communications using a new media proxy leveraging the [Media Transport Library (MTL)](https://github.com/OpenVisualCloud/Media-Transport-Library) and adds the necessary microservices control-plane communications infrastructure to implement any media control protocol.
 
-## Media Proxy
-
-The Media Proxy component in this project is responsible for transporting audio and video stream media data. It handles the routing and forwarding of media data between mesh nodes, ensuring low latency and efficient resource utilization.
-
-It serves as a Data Plane component in a media application's Service Mesh. Media Proxy provides Shared Memory APIs to abstract the complexity of media transport.
-
-More information about the Media Proxy component can be found in the [media-proxy](media-proxy) subdirectory.
-
-### Key Features
 
-- Enables zero memory copy and ultra-low-latency media transfers between containers.
-- Supports transport of video streams via compressed (JPEG XS) or RAW (uncompressed) protocols.
-- Offers compatibility with multiple media transport protocols (SMPTE ST 2110, RTSP, ...).
+## 1. Media Communications Mesh:
+-	Framework Introduction (MCM): is a framework designed for low-latency media transport. It caters specifically to Edge, Edge-to-Cloud, and Cloud environments, connecting media microservices.
+-	Purpose: Its main role is to establish a secure media data plane for inter-microservices communication. This ensures media can be reliably transmitted across different cloud environments.
+-	Technological Basis: The Media Transport Library (MTL) helps form the media data plane, and a control-plane communications infrastructure adds support for media control protocols.
 
-### MemIF
+✅ Key point: The main goal is to enable secure, fast, and standardized media communication between microservices, regardless of the environment (Edge or Cloud).
 
-The Media Proxy leverages the [MemIF](https://s3-docs.fd.io/vpp/24.02/interfacing/libmemif/index.html) protocol for implementing shared memory buffer communications.
+## 2. Media Proxy Introduction:
+-	Core Functionality: The Media Proxy handles the routing and forwarding of media data, specifically audio and video streams, between mesh nodes.
```

Comment:
the line breaks do not work here also (before bullets)

Reply:
fixed